entitybroker/core-providers/src/java/org/sakaiproject/entitybroker/providers/UserEntityProvider.java

```diff
@@ -241,7 +241,9 @@ public void updateEntity(EntityReference ref, Object entity, Map<String, Object>
             edit.setEmail(u.getEmail());
             edit.setFirstName(u.getFirstName());
             edit.setLastName(u.getLastName());
-            edit.setPassword(u.getPassword());
+            if (u.getPassword() != null) {
```

Comment:
I don't think createEntity needs fixing .... the problem I'm trying to solve is that updating a user property nulls out their password ....